llama_hub/semanticscholar/base.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from semanticscholar import SemanticScholar
```

Comment:
could you do the import from the class constructor or methods? The reason is that we want this to be lazily imported. 

Take a look at other loader implementations as example!

Reply:
thank you for this suggestion! I have moved it here https://github.com/emptycrown/llama-hub/pull/439/commits/977671c4763adf24f307b4be55b9b34b667441b8